pallets/messages/src/runtime-api/src/lib.rs

```diff
@@ -23,6 +23,12 @@ use frame_support::inherent::Vec;
 // runtime files (the `runtime` folder)
 sp_api::decl_runtime_apis! {
 
+	/// Runtime Version for Messages
+	/// - MUST be incremented if anything changes
+	/// - Also update in js/api-augment
+	/// - See: https://paritytech.github.io/polkadot/doc/polkadot_primitives/runtime_api/index.html
+	#[api_version(1)]
```

Comment:
This is the default, but wanted to add the documentation and make it more obvious to change if we change the Runtime API